src/voting_nodes.rs

```diff
@@ -0,0 +1,214 @@
+//! The `vote_stage` nodes on the `last_id` of the bank at a regular cadence
+
+use byteorder::{LittleEndian, ReadBytesExt};
+use counter::Counter;
+use crdt::{Crdt, NodeInfo};
+use hash::Hash;
+use log::Level;
+use signature::Pubkey;
+use std::collections::HashMap;
+use std::io::Cursor;
+use std::sync::atomic::AtomicUsize;
+use std::sync::{Arc, RwLock};
+use timing::timestamp;
+use transaction::Vote;
+
+pub struct VotingNodes {
+    pub nodes: HashMap<Pubkey, u64>,
+}
+
+fn make_debug_id(key: &Pubkey) -> u64 {
```

Comment:
I think we can dump the `debug_id` in favor of the formatter we have now.  that would be a good first issue for someone external.

Reply:
I'll file an issue and tag it.

Reply:
https://github.com/solana-labs/solana/issues/1021